src/LobbyForm.tsx

```diff
@@ -79,26 +109,36 @@ const hostStyles = StyleSheet.create({
     fontSize: "8px",
   },
   button: {
+    padding: "0px",
+    boxSizing: "border-box",
     height: "100%",
     border: "none",
-    borderRadius: "0 8px 8px 0",
+    borderRadius: "60px",
     minWidth: "32px",
-    backgroundColor: "#EEEEEE",
+    backgroundColor: "#1D1E33",
+    ":hover": {
+      backgroundColor: "#069AB4",
+    },

```

Comment:
did you have specs for the hover & active states? If not, let's avoid it for now, can just leave it without the hover & active states.

Reasoning:
 If we're using components that are supposed to be like standard components, it would make sense that they would have the feature on them instead.